src/Features/Core/Portable/UnusedReferences/IUnusedReferenceAnalysisService.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Remote;
+
+namespace Microsoft.CodeAnalysis.UnusedReferences
+{
+    internal interface IUnusedReferenceAnalysisService : IWorkspaceService
+    {
+        Task<ImmutableArray<ReferenceInfo>> GetUnusedReferencesAsync(
```

Comment:
ReferenceInfo seems like a simple data type.  I don't think we need SerializableReferenceInfo.  We can just add [DataContract[ to ReferenceInfo.